openapi/openapi.yaml

```diff
@@ -3087,6 +3090,18 @@ components:
               datetime:
                 type: string
                 example: 2021-02-23T09:29:45.254525
+              quay_repo_url:
+                type: string
+                nullable: true
+                example: https://quay.io/thoth-station/s2i-custom-notebook:v0.2.0
+              image_analysis_url:
+                type: string
+                nullable: true
+                example: https://khemenu.thoth-station.ninja/api/v1/analyze/package-extract-220106085109-984feaa8a3862285
```

Comment:
We have multiple deployments. Having URLs that point to one deployment will be misleading. Can this URL be also constructed on frontend? In the worst case, the backend can respond with relative URLs.

Reply:
Yes, these urls can be constructed with only using the analysis id, however it was my understanding that the original issue requested clickable links in the response.

Reply:
Oh, sorry for that. It was meant for Thoth Search. The reference for image analysis could be valuable in the response. Could it be returned without a direct URL? Ex. `/api/v1/analyze/package-extract-220106085109-984feaa8a3862285`. This way, the response will be agnostic to deployment and it could work similarly as pagination.